Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -91,12 +91,6 @@ public override void Initialize()
 
         private void InitializeInternal()
         {
-            if (CameraCache.Main.transform.parent == null)
```

Comment:
Please revert. 

Manager's should take care of setting up what they need, if it's not done so for them already.

Reply:
Or at least call the property to make sure it creates what it needs.